docs: Add reverse proxy X-Forwarded-Port blurb (#33)
grimmory-tools/grimmory-docs: docs/integration/kobo.md

large_client_header_buffers 8 32k;
```

> :::info[X-Forwarded-Port] It's common in reverse proxy setups to port forward public listening ports to a different internal port. If your public listening port is different from $server_port, you must set X-Forwarded-Port to the public listening port. ie. public exposed port is 443 while nginx is listening on 8443

Reviewer: ```suggestion
:::info[X-Forwarded-Port]
If your public listening port is different from your internal $server_port, you should set X-Forwarded-Port to the public listening port.
For example, if your public server is available on port 443, but nginx is listening on 8443 you will need to set X-Forwarded-Port to port 443.
```

Thoughts on this rewording?  I also added a newline.  I don't know if that's needed but it's how the other alert boxes are.
